test/metabase/server/middleware/session_test.clj

```diff
@@ -12,16 +13,15 @@
    [metabase.models :refer [PermissionsGroupMembership Session User]]
    [metabase.public-settings :as public-settings]
    [metabase.public-settings.premium-features :as premium-features]
-   [metabase.public-settings.premium-features-test
-    :as premium-features-test]
+   [metabase.public-settings.premium-features-test :as premium-features-test]
    [metabase.server.middleware.session :as mw.session]
    [metabase.test :as mt]
    [metabase.util.i18n :as i18n]
    [ring.mock.request :as ring.mock]
    [toucan.db :as db])
   (:import
-   (clojure.lang ExceptionInfo)
-   (java.util UUID)))
+   clojure.lang.ExceptionInfo
+   java.util.UUID))
```

Comment:
The old way was correct -- see https://stuartsierra.com/2016/clojure-how-to-ns.html#vectors-or-lists-1